### PR TITLE
NodeOperator List

### PR DESCRIPTION
**Pull request by caseyturnerarrington**
Uploaded Node Operator Info to docs site

### CODE REVIEW

File: docs/general/NodeOperatorList.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# List of available Node Operators
+
+## Node Operator list
+| Sifchain Address | Discord | Other blockchains I run Nodes for | How long do I plan on running a node? | Why would I stop running my node? | What kind of emergency response plans do I have? |
+|:-----------------|:--------|:----------------------------------|:--------------------------------------|:----------------------------------|:-------------------------------------------------|
+sif14tm9600fx088jw55gypcwkwh04j34e9jp68t8r
```
Comment (utx0) [inline docs/general/NodeOperatorList.md:6]: whats this all about?

Comment (caseyturnerarrington) [reply docs/general/NodeOperatorList.md:6]: @utx0 I am going to upload all node validator info here for people to reference when delegating. Still very much a WIP right now.

**Comment by caseyturnerarrington** [comment]
@utx0 this is good to be reviewed now.

**Review by utx0 — APPROVED** [review]

**Review by utx0 — CHANGES_REQUESTED** [review]
Arent these addresses from the `merry-go-round` network?

**Comment by caseyturnerarrington** [comment]
@utx0 this represents all info provided to us by the node validators when asked about providing this info for our MainNet launch. They would use the same addresses as TestNet on MainNet (ideally and they indicated that they would do so)

**Review by taylorrolfe — APPROVED** [review]

**Review by utx0 — APPROVED** [review]

**Pull request merged**